src/wp-includes/theme.php

```diff
@@ -1498,14 +1498,21 @@ function header_image() {
  * @return array
  */
 function get_uploaded_header_images() {
+	$stylesheet    = get_option( 'stylesheet' );
```

Comment:
Why not `get_stylesheet()`?


```suggestion
	$stylesheet    = get_stylesheet();
```